packages/lib-react-components/src/Modal/Modal.js

```diff
@@ -52,6 +57,10 @@ Modal.propTypes = {
     Determines whether the modal is visible or not.
   */
   active: PropTypes.bool,
+  /**
+    The background color for the modal body. It can be set to any CSS color value or color string value from the Zooniverse Grommet theme or an object setting the color for the light and dark theme.
+  */
```

Comment:
Just noting relation to this Issue: https://github.com/zooniverse/front-end-monorepo/issues/1798